package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "reactiflux.com",
-  "version": "π",
+  "version": "0.3.14",
```

Comment:
Netlify local dev tool for testing lambda functions choked hard on this fake version, needed a real actual semver version (despite not using that for anything I can actually discern)